ow_power_system/src/PowerSystemNode.cpp

```diff
@@ -264,15 +283,19 @@ void PowerSystemNode::initAndRun()
     // corresponding prognoser via its message bus.
     // Save the current average mechanical power, in case jointStatesCb runs
     // while processing which would update m_mean_mechanical_power mid-loop.
-    double mech_power = m_mean_mechanical_power;
+    const double actual_mech_power
+      = m_mean_mechanical_power / m_mechanical_efficiency;
+    const double shared_power_load
+      = (
+          actual_mech_power + m_electrical_power + m_power_camera_controller
+          + electricalPowerLights()
+        ) / (NUM_MODELS - m_deactivated_models);
```

Comment:
I haven't seen where the baseline power values are included - am I missing something?

Reply:
Those baseline powers are only present in the PrognoserInputHandler, which is exactly the problem I was pointing out. They aren't being distributed across all batteries. 

My understanding was that I would address that in a PR following this PR.

Reply:
Got it.  That sounds fine -- it can be a hotfix.